examples/launchpad/README.md

```diff
@@ -0,0 +1,40 @@
+# LaunchPad
+
+Users of the Launchpad product will need to pass in certain headers in order to make API requests.
+
+## EdgeHeaders
+EdgeHeaders can be passed into request calls, the additional parameter is available in all reference client functions,
+get_aggs, and snapshots
+
+###### X-Polygon-Edge-ID
+...[DESCRIPTION PENDING]
```

Comment:
Add:
edge_id: is a required Launchpad header. It identifies the Edge User requesting data
edge_ip_address: is a required Launchpad header. It denotes the originating IP Address of the Edge User
edge_user: is an optional Launchpad header. It denotes the originating UserAgent of the Edge User requesting data.